src/components/data-exploration/CellInfo.jsx

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
-import _ from 'lodash';
 import { Card } from 'antd';
 import PropTypes from 'prop-types';
 
+const EM = 16; // px
+const CELLINFO_WIDTH = 200; // px
```

Comment:
Moving `CELLINFO_WIDTH`to `CellInfo` is definitely better. Can you get rid of this hardcoded width variable entirely and have another way to know when to invert or not? Or another way to detect the width? The width of the cellinfo depends on the size of the text in its content and hardcoding it is not ideal.

Reply:
The way I thought of is by counting the length of each of the lines that is going to be displayed in the box, and then multiply it by the average width of a character. Add left and right padding to that, and we should get the width of the box.

Reply:
I tried using `ref` but it turns out there are issues using refs in Antd components: https://github.com/ant-design/ant-design/issues/28582. Hence, we can't get a hold of the rendered component to get it's width and height.

Reply:
I tried using a ref to get the element and then accessing its width and height using `.offsetWidth` and `.offsetHeight`. These return 0 in Chrome because of some optimizations that [Chrome does](https://stackoverflow.com/a/70043254/1940886).